data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3360 @@
+			`	"Well if you want to work with us then you will have to set that preferance aside. And do not forget the translator is for communicating with us as well. Some among the most conversative of us despise speaking in your language... This will be a token of good faith on your part."`
+			choice
+				`	"I guess I will try to land on their planets."`
+					accept
+				`	"I will not work for you anymore then."`
+					decline
+	
+	waypoint "Chy'chra"
+	waypoint "Pik'ro'iyak"
+	waypoint "Ik'kara'ka"
+	waypoint "Chirr'ay'akai"
+	waypoint "Sich'ka'ara"
+	waypoint "Kiro'ku"
+	waypoint "Ka'pru"
+	waypoint "Ap'arak"
+	waypoint "Ka'ch'chrai"
+	waypoint "Ek'kek'ru"
+	waypoint "Si'yak'ku"
+	waypoint "Iyech'yek"
+	waypoint "Kiru'kichi"
+	waypoint "Es'sprak'ai"
+	
+	on waypoint
+		dialog `You have scouted all of the Wanderer's systems. If you can somehow speak Hai already, it is time to come back to <destination>, for a reward.`
+	to complete
+		has "language: Wanderer"
+	on complete
+		log `The Unfettered asked me to be able to speak like a True Hai, because that is the language in which I should make demands to the Unfettered later.`
+		log "Wanderer" `The Wanderers are a birdlike specie that seems weak and is said to inhabit the old lands of the Unfettered Hai.`
+		payment 1000000
+		dialog `You find the herald back. "Now you can speak to us like a True Hai, finally. By respect to you, we will keep speaking to you in your language. Except for some maybe... You only should use the translator to speak and understand our elders, or at least some of them. They will probably do as if they noticed nothing, but trust me, they care. Oh and here are <payment> for the intel, we could not have gotten it so easily given they shoot our ships on sight."`
+
+
+
+mission "Unfettered: Bring Detainer"
+	landing
+	name `Recover a detainer`
+	description `He Yi has asked you to earn a translation device by bringing back a powerful alien weapon, said to be able to neutralize the engines of its target. You should probably look around the sestor automaton's space.`
+	source "Warfeed"
+	to offer
+		has "Unfettered: Know Your Enemy: active"
+		has "First Contact: Wanderer: offered"
+		not "language: Wanderer"
+	on offer
+		conversation
+			`You go find He Yi in his war room, he does not seem too surprised to see you. "I expected you to come here. You most want one of us to act as 'diplomat' with the aliens from the North."`
+			`	You nod.`
+			`	"Well that is not our way, we do not ask, we take. Things need to be earned, in battle or otherwise, else they have no value. I  have obtained a translator, that is able to translate your backwards language to the one of the Hai. Because these birds speak Hai, you will be able to communicate with them later. I was not sure you were unable to speak Hai, or that the birds would not understand you, they seem to have grasped Hai very quickly, trough methods I prefer not to imagine."`
```

Comment:
```suggestion
			`	"Well that is not our way, we do not ask, we take. Things need to be earned, in battle or otherwise, else they have no value. I  have obtained a translator, that is able to translate your backwards language to the one of the Hai. Because these birds speak Hai, you will be able to communicate with them later. I was not sure before that you were still unable to speak Hai, or that the birds would not understand your language, as they seem to have grasped Hai very quickly, through methods I prefer not to imagine."`
```